README.md

```diff
@@ -5,19 +5,17 @@ and password complexity (password must consist capital and small letters, number
 contain the user's name or username). A user lock on defined time period after x number of failed logins is also
 implemented.
 
-This extension is compatible with CKAN 2.8 and 2.9.
+This extension is compatible with CKAN 2.11+.
 
 
 ## Requirements
 
 Compatibility with core CKAN versions:
 
-| CKAN version    | Compatible? |
-| --------------- |-------------|
-| 2.6 and earlier | not tested  |
-| 2.7             | not tested  |
-| 2.8             | Yes         |
-| 2.9             | Yes         |
+| CKAN version | Compatible? |
+|--------------|-------------|
+| 2.10         | not tested  |
+| 2.11         | not tested  |
```

Comment:
I think this should be `Yes`. Also as we are saying so above. `... compatible with CKAN 2.11+`